<ui version="4.0" >
 <class>qslPr</class>
 <widget class="QMainWindow" name="qslPr" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>563</height>
   </rect>
  </property>
  <property name="font" >
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle" >
   <string>QSL - Druck</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" name="gridLayout" >
    <item row="0" column="0" colspan="2" >
     <widget class="QLabel" name="label" >
      <property name="palette" >
       <palette>
        <active>
         <colorrole role="WindowText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>54</red>
            <green>70</green>
            <blue>163</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>54</red>
            <green>70</green>
            <blue>163</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>119</red>
            <green>119</green>
            <blue>119</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font" >
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text" >
       <string>  QSL - Drucken</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="2" >
     <widget class="QLabel" name="label_2" >
      <property name="text" >
       <string>Datentr√§ger :  </string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="4" >
     <widget class="QLabel" name="MediumType" >
      <property name="palette" >
       <palette>
        <active>
         <colorrole role="WindowText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>54</red>
            <green>70</green>
            <blue>163</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>54</red>
            <green>70</green>
            <blue>163</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>119</red>
            <green>119</green>
            <blue>119</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font" >
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text" >
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="5" colspan="5" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>247</width>
        <height>22</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="11" >
     <widget class="Line" name="line" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item rowspan="2" row="2" column="0" >
     <widget class="QLabel" name="label_4" >
      <property name="font" >
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text" >
       <string>   Logbook :</string>
      </property>
     </widget>
    </item>
    <item rowspan="2" row="2" column="1" >
     <widget class="QLabel" name="labelLogbook" >
      <property name="palette" >
       <palette>
        <active>
         <colorrole role="WindowText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>26</red>
            <green>163</green>
            <blue>61</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>26</red>
            <green>163</green>
            <blue>61</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>119</red>
            <green>119</green>
            <blue>119</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font" >
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text" >
       <string>fun</string>
      </property>
     </widget>
    </item>
    <item rowspan="2" row="2" column="2" colspan="5" >
     <widget class="QLabel" name="labelQsos" >
      <property name="text" >
       <string>Qsos</string>
      </property>
     </widget>
    </item>
    <item rowspan="2" row="2" column="7" >
     <widget class="QLabel" name="label_6" >
      <property name="text" >
       <string> Operator :</string>
      </property>
     </widget>
    </item>
    <item rowspan="2" row="2" column="8" colspan="2" >
     <widget class="QLabel" name="labelOperator" >
      <property name="font" >
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text" >
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="3" column="10" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>151</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item rowspan="7" row="4" column="0" colspan="10" >
     <widget class="QTreeWidget" name="prList" >
      <property name="palette" >
       <palette>
        <active>
         <colorrole role="Base" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>245</red>
            <green>255</green>
            <blue>243</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Base" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>245</red>
            <green>255</green>
            <blue>243</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Base" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="focusPolicy" >
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="rootIsDecorated" >
       <bool>false</bool>
      </property>
      <column>
       <property name="text" >
        <string>Call</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>Via Mgr</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>QSL s</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>QSL r</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>Band</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>Datum</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>Start</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>Mode</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>TxPwr</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>Rig</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>id</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="4" column="10" >
     <widget class="QLabel" name="label_3" >
      <property name="text" >
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="10" >
     <widget class="QLabel" name="label_10" >
      <property name="font" >
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text" >
       <string> Drucke ab Label-Nr</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="10" >
     <widget class="QFrame" name="frame" >
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QPushButton" name="Lab_1" >
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QPushButton" name="Lab_5" >
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string>5</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QPushButton" name="Lab_2" >
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string>2</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QPushButton" name="Lab_6" >
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string>6</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QPushButton" name="Lab_3" >
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string>3</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QPushButton" name="Lab_7" >
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string>7</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QPushButton" name="Lab_4" >
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string>4</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QPushButton" name="Lab_8" >
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string>8</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="7" column="10" >
     <widget class="QLabel" name="Labtype" >
      <property name="text" >
       <string>T</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="10" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>88</width>
        <height>69</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="9" column="10" >
     <widget class="QCheckBox" name="checkBoxForm" >
      <property name="text" >
       <string>Erzeuge PS-format</string>
      </property>
     </widget>
    </item>
    <item row="10" column="10" >
     <widget class="QLabel" name="labelPrinter" >
      <property name="text" >
       <string/>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="10" >
     <widget class="QLabel" name="label_5" >
      <property name="palette" >
       <palette>
        <active>
         <colorrole role="WindowText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>30</red>
            <green>123</green>
            <blue>107</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>30</red>
            <green>123</green>
            <blue>107</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>119</red>
            <green>119</green>
            <blue>119</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font" >
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text" >
       <string>    Clicke in die Vorschlagsliste um den Druckvorschlag zu entfernen</string>
      </property>
     </widget>
    </item>
    <item row="11" column="10" >
     <widget class="QPushButton" name="ButtonPrinter" >
      <property name="font" >
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="focusPolicy" >
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text" >
       <string>Drucken</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0" >
     <widget class="QLabel" name="label_9" >
      <property name="font" >
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text" >
       <string>  Suche Call  </string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="12" column="1" colspan="2" >
     <widget class="QLineEdit" name="CallEdit" />
    </item>
    <item row="12" column="3" colspan="3" >
     <widget class="QCheckBox" name="checkBoxOrder" >
      <property name="focusPolicy" >
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text" >
       <string>Sortiere n. Datum</string>
      </property>
     </widget>
    </item>
    <item row="12" column="6" colspan="3" >
     <widget class="QCheckBox" name="checkBoxSqsl" >
      <property name="focusPolicy" >
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text" >
       <string>Einzel-Best√§tigung</string>
      </property>
     </widget>
    </item>
    <item row="12" column="9" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>285</width>
        <height>27</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="12" column="10" >
     <widget class="QCheckBox" name="checkBoxLpr" >
      <property name="focusPolicy" >
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text" >
       <string>Drucke auf Lineprinter</string>
      </property>
      <property name="checked" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="13" column="0" colspan="11" >
     <widget class="QPushButton" name="ButtonEnde" >
      <property name="palette" >
       <palette>
        <active>
         <colorrole role="Button" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>175</red>
            <green>215</green>
            <blue>217</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Button" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>175</red>
            <green>215</green>
            <blue>217</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Button" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>175</red>
            <green>215</green>
            <blue>217</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font" >
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="focusPolicy" >
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text" >
       <string> F10   -  ENDE</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>838</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei" >
    <property name="title" >
     <string>Datei         </string>
    </property>
    <addaction name="actionENDE" />
   </widget>
   <widget class="QMenu" name="menuConfiguriere_Druckermedium" >
    <property name="title" >
     <string>Configuriere Druckermedium              </string>
    </property>
    <addaction name="actionPrMedium" />
    <addaction name="separator" />
    <addaction name="actionLabel" />
    <addaction name="actionLabelSammel" />
    <addaction name="separator" />
    <addaction name="actionKarte" />
    <addaction name="actionKarteSammel" />
    <addaction name="separator" />
    <addaction name="actionKarteSwl" />
    <addaction name="actionLabelSwl" />
   </widget>
   <widget class="QMenu" name="menuInfo" >
    <property name="title" >
     <string>Hilfe              </string>
    </property>
    <addaction name="actionHilfe" />
   </widget>
   <addaction name="menuDatei" />
   <addaction name="menuConfiguriere_Druckermedium" />
   <addaction name="menuInfo" />
  </widget>
  <action name="actionENDE" >
   <property name="icon" >
    <iconset resource="../qtlogRessourcen.qrc" >
     <normaloff>:/new/prefix1/icon/cancel.png</normaloff>:/new/prefix1/icon/cancel.png</iconset>
   </property>
   <property name="text" >
    <string>ENDE        </string>
   </property>
  </action>
  <action name="actionPrMedium" >
   <property name="text" >
    <string>Auswahl: Label / Karten</string>
   </property>
  </action>
  <action name="actionLabel" >
   <property name="text" >
    <string>Label</string>
   </property>
  </action>
  <action name="actionLabelSwl" >
   <property name="text" >
    <string>Label-SWL</string>
   </property>
  </action>
  <action name="actionKarte" >
   <property name="text" >
    <string>Karte</string>
   </property>
  </action>
  <action name="actionKarteSwl" >
   <property name="text" >
    <string>Karte-SWL</string>
   </property>
  </action>
  <action name="actionLabelSammel" >
   <property name="text" >
    <string>Label-SammelQsl</string>
   </property>
  </action>
  <action name="actionKarteSammel" >
   <property name="text" >
    <string>Karte-SammelQsl</string>
   </property>
  </action>
  <action name="actionHilfe" >
   <property name="text" >
    <string>Qsl Drucken</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../qtlogRessourcen.qrc" />
 </resources>
 <connections/>
</ui>
